_studio/mfx_lib/encode_hw/vp9/src/mfx_vp9_encode_hw.cpp

```diff
@@ -75,7 +75,7 @@ mfxStatus MFXVideoENCODEVP9_HW::Query(mfxCoreInterface *core, mfxVideoParam *in,
                     {
                         mfxExtBuffer *pCorrectedBuf = GetExtendedBuffer(toValidate.ExtParam, toValidate.NumExtParam, pInBuf->BufferId);
                         MFX_CHECK_NULL_PTR1(pCorrectedBuf);
-                        memcpy_s(pOutBuf, pOutBuf->BufferSz, pCorrectedBuf, pCorrectedBuf->BufferSz);
+                        *pOutBuf = *pCorrectedBuf;
```

Comment:
It copies only buffer header, but not complete buffer!

Reply:
Yes, thanks, my merge error

Reply:
I had a closer look at this comment.
Not sure what do u mean?

Reply:
std::copy(pCorrectedBuf, pCorrectedBuf + pCorrectedBuf->BufferSz, pOutBuf);

Reply:
you need to cast it to byte pointers first, then you can do std::copy